.github/workflows/ci.yml

```diff
@@ -234,10 +234,6 @@ jobs:
     name: Unit Testing
     needs: test-import
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        image-tag: [v22.2.0, v23.1.0]
```

Comment:
Removed the matrix as we will now run codegen for 22.2 and 23.1 within same job/runner to generate a combined package.